src/protocols/login.js

```diff
@@ -224,45 +239,44 @@ function verifyIdenAssertV01(nonceDB: NonceDB, origin: string,
 
 /**
  * Verify a signed packet
- * @param {Object} nonceDB
- * @param {String} origin
  * @param {Object} jwsHeader
  * @param {Object} jwsPayload
  * @param {Buffer} signatureBuffer
- * @returns {?NonceVerified} nonce
+ * @returns {boolean} result
  */
-function verifySignedPacketV01(nonceDB: NonceDB, origin: string, jwsHeader: JwsHeader, jwsPayload: JwsPayload,
-  signatureBuffer: Buffer): ?NonceVerified {
+function verifySignedPacketV01(jwsHeader: JwsHeader, jwsPayload: JwsPayload,
```

Comment:
`verifyIdenAssertV01` is for verifying `IDENASSERTV01` specific data inside packet. It requires to call `verifysignedPacket` before to verify packet signature. We could include `verifysignedPacket` into `verifyIdenAssertV01`

Reply:
I've added the functions:
- verifySignedPacketGeneric,
- verifySignedPacketIdenAssert,

and:
- signGenericSigV01,
- signIdenAssertV01,